docs/source/conf.py

```diff
@@ -410,7 +411,7 @@ def generate_weights_table(module, table_name, metrics, dataset, include_pattern
 )
 generate_weights_table(
     module=M.quantization,
-    table_name="classification_quant",
+    table_name="quantized_classification",
```

Comment:
I feel that `Quantized Classification Weights Table` looks better than `Classification Quant Weights Table` as a title. 

Does this sound okay?